EF/DeliveryService/DeliveryService/LinqTask/FoodManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using DeliveryService.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DeliveryService.LinqTask
+{
+   public class FoodManager
+    {
+        public IList<Food> Foods { get; set; }
+        public IList<FoodType> FoodTypes { get; set; }
+        public IList<Manufacturer> Manufacturers { get; set; }
+
+        public FoodManager(IList<Manufacturer> manufacturers, IList<FoodType> foodTypes, IList<Food> foods)
+        {
+            Foods = foods;
+            FoodTypes = foodTypes;
+            Manufacturers = manufacturers;
+        }
+
+        public IEnumerable GetSortFoods()
```

Comment:
better to return some more specific then IEnumerable
IEnumerable<Food>
in other places as well